contents/docs/libraries/js/_snippets/install.mdx

```diff
@@ -1,39 +1,11 @@
-import Snippet from "../../../integrate/\_snippets/snippet.mdx"
+import Snippet from "../../../integrate/\_snippets/install-js-snippet.mdx"
 import Badge from "../../../getting-started/\_snippets/badge.mdx"
+import InstallJSViaNPM from "../../../integrate/\_snippets/install-js-via-npm.mdx"
 
 ### Option 1: Add JavaScript snippet to your HTML <Badge>Recommended</Badge>
```

Comment:
Hmm I prefer to have "Option 1" in the text, so that the reader knows ahead of time that there are multiple options, as opposed to "do these steps in this order"